release/Magarena/scripts/Erayo__Soratami_Ascendant.groovy

```diff
@@ -0,0 +1,21 @@
+[
+    new OtherSpellIsCastTrigger() {
+        @Override
+        public MagicEvent executeTrigger(final MagicGame game,final MagicPermanent permanent, final MagicCardOnStack cardOnStack) {
+            return (game.getSpellsCast() == 3) ?
+                new MagicEvent(
+                    permanent,
+                    cardOnStack,
+                    this,
+                    "Whenever the fourth spell of a turn is cast, flip SN."
```

Comment:
trigger condition should not be in the effect text, effect text should be "Flip SN."